features/support/capybara.rb

```diff
@@ -1,117 +1,49 @@
 # frozen_string_literal: true
 
 require 'selenium/webdriver'
-require 'capybara/minitest'
-include Capybara::Minitest::Assertions
 
-# in case firefox is needed!
-#Capybara.register_driver :selenium do |app|
-#  Capybara::Selenium::Driver.new(app, :browser => :firefox)
-#end
-
-DEFAULT_JS_DRIVER = :headless_chrome
-# in case firefox is needed!
-# DEFAULT_JS_DRIVER = :headless_firefox
-
-Capybara.default_driver = :rack_test
-Capybara.javascript_driver = DEFAULT_JS_DRIVER
-Capybara.default_selector    = :css
+# Capybara.default_max_wait_time = 10
 Capybara.disable_animation = true
-
-# Capybara 3 changes the default server to Puma. It can be reverted to the previous default of WEBRick by specifying:
-Capybara.server = :webrick
-
-# see http://www.elabs.se/blog/60-introducing-capybara-2-1
-Capybara.configure do |config|
-  config.default_driver = :rack_test
-  config.javascript_driver = DEFAULT_JS_DRIVER
-  config.raise_server_errors = true
-  config.match = :prefer_exact
-  config.always_include_port = true
-  config.default_max_wait_time = 10
-end
-
-# Needed because cucumber-rails requires capybara/cucumber
-# https://github.com/cucumber/cucumber-rails/blob/7b47bf1dda3368247bf2d45bcb17a224e80ec6fd/lib/cucumber/rails/capybara.rb#L3
-# https://github.com/teamcapybara/capybara/blob/2.18.0/lib/capybara/cucumber.rb#L17-L19
-Before '@javascript' do
-  Capybara.current_driver = DEFAULT_JS_DRIVER
-end
+Capybara.javascript_driver = :headless_chrome
+Capybara.match = :prefer_exact # TODO: this is Capybara v1 default, we should use :one or :smart
```

Comment:
Why not change it to `:one` or `:smart` directly?

Reply:
I tried `:smart` in another PR, but it caused many `Ambiguous match` errors. Our selectors are probably sub-optimal in some occasions, but it's probably an unnecessary effort to fix all of them at once. Maybe we can go little by little.
Test results with `:smart` mode: https://app.circleci.com/pipelines/github/3scale/porta/30206/workflows/cf089510-f40f-42c8-ba6b-d53df680ec69/jobs/338231